muxers/mplex/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.39.0 [unreleased]
+
+- Increased the lenght of Stream ID to 60 bits.
```

Comment:
Please move this line under the `# 0.29.0 [...]` section below the `- Update dependencies` line.